src/containers/Home/homeReducer.js

```diff
@@ -0,0 +1,151 @@
+const initialState = {
+    reminders:[], 
+    error:null, 
+}
+
+
+export default function homeReducer (state = initialState , action ){
+   switch(action.type){
+        case "FETCH_REMINDER_PENDING": {
```

Comment:
For this project (as it is small one) - having action types as string is fine. But redux suggest to declare them as constants and use those constants everywhere. You can find more info here: http://redux.js.org/docs/recipes/ReducingBoilerplate.html. Please go through it.